modules/ROOT/pages/config-workflow.adoc

```diff
@@ -1,18 +1,21 @@
-= Post-installation Tasks
+= Configure Anypoint Platform PCE
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :page-aliases: install-add-tasks.adoc
 
 After you complete Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) installation, review the following tasks, many of which are required before you can successfully deploy and manage applications, register APIs, and use API Exchange on an Anypoint Platform PCE environment running on your own local network.
 
-* If you used the GUI installer, close port 61009 in the cluster security group. If you are using AWS, you can use the AWS Web console to do this.
-* xref:access-management-dns.adoc[Configure DNS on Anypoint Private Cloud]
+* Close port 61009 in the cluster security group. If you are using AWS, you can use the AWS Web console to do this.
```

Comment:
@federicoboerr should the "close port 61009" be removed from post-installation tasks? It is included in the internal installation instructions.